src/java.base/share/classes/java/lang/Class.java

```diff
@@ -554,137 +553,129 @@ static String typeVarBounds(TypeVariable<?> typeVar) {
     }
 
     /**
-     * Returns {@code true} if this class is an inline class.
+     * Returns {@code true} if this class is a primitive class.
      *
-     * @return {@code true} if this class is an inline class
+     * @return {@code true} if this class is a primitive class, otherwise false
      * @since Valhalla
      */
-    public boolean isInlineClass() {
-        return (this.getModifiers() & INLINE) != 0;
+    public boolean isPrimitiveClass() {
+        return (this.getModifiers() & PRIMITIVE_CLASS) != 0;
     }
 
     /**
-     * Returns a {@code Class} object representing the <em>value projection</em>
-     * type of this class if this {@code Class} represents the reference projection
-     * type of an {@linkplain #isInlineClass() inline class}.
-     * If this {@code Class} represents the value projection type
-     * of an inline class, then this method returns this class.
+     * Returns a {@code Optional<Class>} object representing the <em>primitive value type</em>
+     * of this class if this {@code Class} represents the <em>primitive reference type</em>
+     * of a {@linkplain #isPrimitiveClass() primitive class}.
+     * If this {@code Class} represents the primitive value type
```

Comment:
"the primitive reference type of a primitive class" perhaps should be shortened to 
"the reference type of a primitive class"
"the value type of a primitive class" similarly for value type.